services/parsers/symbolscraper/model/predictor.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, config: PredictorConfig):
         Initialize your model using the passed parameters
         """
         self._config = config
-        self._sscraper = SymbolScraperParser("sscraper", print_info=False)
+        self._sscraper = SymbolScraperParser("sscraper", debug=False)
```

Comment:
Here you can call `symbol_scraper_parser.logger.setLevel(logging.DEBUG)` based on config.

Reply:
ok